esp/esp/program/modules/handlers/jsondatamodule.py

```diff
@@ -468,6 +475,7 @@ def class_info(self, request, tl, one, two, module, extra, prog):
     class_info.cached_function.depend_on_model(ClassSection)
 
     @aux_call
+    @no_auth
```

Comment:
Oh, right, I forgot that `wraps` copies `__dict__` in addition to `__name__` and so on.  In that case, consistency would be nice but I don't care that much.